pybop/optimisers/scipy_optimisers.py

```diff
@@ -200,9 +200,11 @@ def base_callback(intermediate_result: Union[OptimizeResult, np.ndarray]):
                 x_best = intermediate_result
                 cost = self.cost(x_best, apply_transform=True)
```

Comment:
Please change `cost` to `cost_best` here for clarity. The cost is logged during the call to the `cost_wrapper`.